.travis.yml

```diff
@@ -6,11 +6,13 @@ services:
   - docker
 
 script:
- - docker-compose build tests
+ - docker-compose build release
```

Comment:
release, protože to je to co to je. Je to image, který se releasne do QUAY hubu.